grammars/gfm.cson

```diff
@@ -15,34 +15,28 @@
     'name': 'constant.character.escape.gfm'
   }
   {
-    'begin': '(?<=^|[^\\w\\d\\*])\\*\\*\\*(?!$|\\*|\\s)'
-    'end': '(?<!^|\\s)\\*\\*\\**\\*(?=$|[^\\w|\\d])'
-    'name': 'markup.bold.italic.gfm'
-  }
-  {
-    'begin': '(?<=^|[^\\w\\d_])___(?!$|_|\\s)'
-    'end': '(?<!^|\\s)___*_(?=$|[^\\w|\\d])'
-    'name': 'markup.bold.italic.gfm'
-  }
-  {
-    'begin': '(?<=^|[^\\w\\d\\*])\\*\\*(?!$|\\*|\\s)'
-    'end': '(?<!^|\\s)\\*\\**\\*(?=$|[^\\w|\\d])'
-    'name': 'markup.bold.gfm'
-  }
-  {
-    'begin': '(?<=^|[^\\w\\d_])__(?!$|_|\\s)'
-    'end': '(?<!^|\\s)__*_(?=$|[^\\w|\\d])'
-    'name': 'markup.bold.gfm'
+    'match': '^___([\\s\\S]+?)___(?!_)|^\\*\\*\\*([\\s\\S]+?)\\*\\*\\*(?!\\*)'
```

Comment:
I currently have `'match': '(?<=^|[^\\w\\d\\*])(\\*\\*\\*)([\\s\\S]+?)(\\*\\*\\*)(?=$|[^\\w|\\d])'`, and I [the tokens are](https://github.com/atom/language-gfm/blob/79b391a22d17da19cd7db4e475cfa1147157ebab/spec/gfm-spec.coffee#L38):

```json
[
    {
        "value": "this is ",
        "scopes": [
            "source.gfm"
        ],
        "bufferDelta": 8,
        "hasPairedCharacter": false,
        "screenDelta": 8
    },
    {
        "value": "***bold italic***",
        "scopes": [
            "source.gfm",
            "markup.bold.italic.gfm"
        ],
        "bufferDelta": 17,
        "hasPairedCharacter": false,
        "screenDelta": 17
    },
    {
        "value": " text",
        "scopes": [
            "source.gfm"
        ],
        "bufferDelta": 5,
        "hasPairedCharacter": false,
        "screenDelta": 5
    }
]
```

I expected that by doing `(\\*\\*\\*)` these will be grouped in another element (`{ value: "***" }`). The regex101 tester works fine in this direction:

![image](https://cloud.githubusercontent.com/assets/2864371/7200099/4b9a1258-e502-11e4-89a4-c472cb45a6d9.png)

How can I catch these characters into a separate group when using `apm test`? Also, what tool is used for parsing this regex?